packages/core/src/families/casper/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { z } from "zod";
+import type { TransactionCommon } from "../index";
+import type { schemaRawCasperTransaction } from "./validation";
+import type BigNumber from "bignumber.js";
+
+export type CasperTransaction = TransactionCommon & {
+  readonly family: RawCasperTransaction["family"];
+  fees: BigNumber;
```

Comment:
`fees` is optional on all other families, is this expected ?
```suggestion
  fees?: BigNumber;
```